packages/legend-studio/src/components/editor/edit-panel/uml-editor/AssociationEditor.tsx

```diff
@@ -159,14 +170,19 @@ const AssociationPropertyBasicEditor = observer(
 
     return (
       <div className="property-basic-editor">
-        <input
-          className="property-basic-editor__name"
-          disabled={isReadOnly}
-          value={property.name}
-          spellCheck={false}
-          onChange={changeValue}
-          placeholder={`Property name`}
-        />
+        <div className="input-group__input property-basic-editor__input">
```

Comment:
why do we still need this `wrapper` `<div>`?

Reply:
Oh that is have a margin to the right of the input box. Same at other places too. Could have added it to the `Input.tsx` but I think that will mess the styling at other places

Reply:
then we should add the style to the `class` you set for the `<input>` right?